resources/config.py

```diff
@@ -1,3 +1,7 @@
 from os import path
```

Comment:
I was under the impression that you were trying to detect new models automatically, if they are in the folder. But this is fine. So, in this case we just will have to add the name of the model in the list right?

Reply:
Yeah, I wanted to do something like checking for a new files by Python script, but haven't figured out what will be the good way to import them. So I went for explicitly making a list of models and you can add them to the list.